discovery_engine_core/bindings/src/types/init_config.rs

```diff
@@ -0,0 +1,136 @@
+// Copyright 2022 Xayn AG
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, version 3.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+//! FFI functions for handling [`InitConfig`] instances.
+
+use std::ptr::addr_of_mut;
+
+use core::{InitConfig, Market};
+
+/// Returns a pointer to the `api_key` field of a document.
```

Comment:
same for all docs in this module
```suggestion
/// Returns a pointer to the `api_key` field of a configuration.
```